atlasdb-commons/build.gradle

```diff
@@ -12,7 +12,7 @@ dependencies {
     compile project(":commons-executors")
     compile 'javax.ws.rs:javax.ws.rs-api:2.0.1'
     compile group: 'com.fasterxml.jackson.core', name: 'jackson-annotations'
-    compile group: 'io.dropwizard.metrics', name: 'metrics-core'
+    compile group: 'com.codahale.metrics', name: 'metrics-core'
```

Comment:
we should exclude `com.codahale.metrics` in `atlasdb-timelock-server/build.gradle` and `atlasdb-dropwizard-bundle/build.gradle`